app/src/main/java/com/greenfox/gitinder/api/reciever/BackgroundReceiver.java

```diff
@@ -40,6 +46,12 @@ public void onReceive(Context context, Intent intent) {
             @Override
             public void onResponse(Call<Matches> call, Response<Matches> response) {
                 Log.d(TAG, "onResponse: matches called");
+                if (response.body() != null && sharedPreferences.getBoolean(Constants.ENABLE_NOTIFICATIONS,false)){
+                    for (Match m: response.body().getMatches()) {
+                        notificationService.pushNewMatchNotification(m,context);
```

Comment:
@WimmermM only update the match service with new matches here, in match service if there is new match, push noti for the new ones